crescent/internal/app_command.py

```diff
@@ -81,10 +80,11 @@ class AppCommand(CommandBuilder):
     type: CommandType
     name: str
     guild_id: Snowflakeish | None
-    default_permission: UndefinedOr[bool]
 
     description: str | None = None
     options: Sequence[CommandOption] | None = None
+    default_member_permissions: UndefinedType | int | Permissions = UNDEFINED
+    is_dm_enabled: bool = True
```

Comment:
Used a different name than the kwargs for commands. Seems inconsistent to me.

Reply:
Nevermind I see that this is used by hikari

Reply:
The worst part is hikari's wording is different than the Discord API wording